example/README.md

```diff
@@ -0,0 +1,342 @@
+
+3. You can use [VSCode with Aqua extension](https://marketplace.visualstudio.com/items?itemName=FluenceLabs.aqua) for syntax highlighting  and better developer experience.
+
+## Add echo service in rust
+
+0. If you have **Apple Silicon** you should install cargo manually:
+
+    ```bash
+    # for m1:
+    cargo install marine --version '0.12.1' --root ~/.fluence/cargo
+    ```
+
+1. Add echo service:
+
+    ```bash
+    fluence service add echoService
+    ```
+
+    output:
+
+    ```bash
+    $ fluence service add echoService
+    Added echoService to fluence.yaml
+    ```
+
+2. Following [code](echoService/modules/echoService/src/main.rs) returns peer id of the host and initial message:
+
+    ```rust
+    #[marine]
+    pub fn echo(msg: String) -> String {
+        format!("{}: {}", marine_rs_sdk::get_call_parameters().host_id, msg)
+    }
+    ```
+
+
+### Deploy service and test it
+
+1. Deploy service:
+
+    ```bash
+    fluence deploy
+    ```
+
+    Output, peer ids and service ids may differ:
+
+    ```bash
+    $ fluence deploy
+    Making sure all services are downloaded... done
+    Making sure all modules are downloaded and built... done
+
+    Going to deploy services described in ~/Documents/dev/fluencelabs/registry/example/fluence.yaml:
+
+    echoService:
+    get: echoService
+    deploy:
+        - deployId: default
+
+
+    ? Do you want to deploy all of these services? Yes
+    Deploying:
+    service: echoService
+    deployId: default
+    on: 12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS
+    ... done
+    Compiling ~/Documents/dev/fluencelabs/registry/example/.fluence/aqua/deployed.app.aqua... done
+
+    Currently deployed services listed in ~/Documents/dev/fluencelabs/registry/example/.fluence/app.yaml:
+
+    echoService:
+    default:
+        - blueprintId: fd21793bae96d1519599346591bfe7ea75614d9fe7f7c824b279e5a2dc841ee0
+        serviceId: 785487e4-de08-4fa4-bdfd-df981ec66da7
+        peerId: 12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS
+    ```
+
+
+2.  The following [code](src/aqua/main.aqua) declares module, imports App and EchoService and defines method to call `echo`:
+
+    ```rust
+    module Main
+
+    import App from "deployed.app.aqua"
+    import EchoService from "services/echoService.aqua"
+    export App, echo
+
+    func echo(msg: string) -> string:
+        services <- App.services()
+
+        on services.echoService.default[0].peerId:
+            EchoService services.echoService.default[0].serviceId
+            res <- EchoService.echo(msg)
+        <- res
+    ```
+
+3. Let’s test our echo service:
+
+    ```bash
+     fluence run -f 'echo("hi")'
+    ```
+
+    Output:
+
+    ```bash
+    $ fluence run -f 'echo("hi")'
+    Running:
+      function: echo("hi")
+      relay: /dns4/kras-08.fluence.dev/tcp/19001/wss/p2p/12D3KooWFtf3rfCDAfWwt6oLZYZbDfn9Vn7bv7g6QjjQxUUEFVBt
+    ... done
+
+    Result:
+
+    "12D3KooWCMr9mU894i8JXAFqpgoFtx6qnV1LFPSfVc3Y34N4h4LS: hi"
+    ```
+
```

Comment:
A short summary for the first part of the example would be great. Something like: "We've successfully build and deployed a service written in Rust, and called it from our Aqua code using the `fluence` CLI"